jdk/test/sun/security/ssl/X509TrustManagerImpl/distrust/Camerfirma.java

```diff
@@ -21,18 +21,20 @@
  * questions.
  */
 
+import javax.net.ssl.X509TrustManager;
 import java.io.File;
 import java.security.Security;
-import java.time.*;
-import java.util.*;
-import javax.net.ssl.*;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Date;
 
-/**
+/*
  * @test
- * @bug 8346587
+ * @bug 8346587 8350498
  * @summary Check that TLS Server certificates chaining back to distrusted
- *          Camerfirma roots are invalid
- * @library /lib/security
+ *          Camerfirma root are invalid
+ * @library /test/lib
```

Comment:
The `roots` => `root` change is fine. The test library change not. 8u has `lib/security` not `/test/lib`.

Reply:
Yep, uploaded the changes in the recent commit. Sorry by that.